pepper-apis/dss-core/src/main/java/org/broadinstitute/ddp/model/activity/definition/TabularBlockDef.java

```diff
@@ -44,6 +45,15 @@ public int getRowsCount() {
 
     @Override
     public Stream<QuestionDef> getQuestions() {
-        return getRows().stream().map(TabularRowDef::getQuestions).flatMap(Collection::stream);
+        List<TabularRowDef> rowDefs = getRows().stream().collect(Collectors.toList());
+        List<QuestionDef> questionDefList = new ArrayList<>();
+        for (TabularRowDef rowDef : rowDefs) {
+            for (QuestionBlockDef colDef: rowDef.getQuestions()) {
+                if (colDef != null) {
+                    questionDefList.add(colDef.getQuestion());
+                }
+            }
+        }
+        return questionDefList.stream();
```

Comment:
Instead of adding 9 new lines of code, I'd suggest extending the original stream statement with notNull call